docs/maintenance.md

```diff
@@ -0,0 +1,81 @@
+Maintenance guide
+=================
+
+This is the maintenance guide for pvc-autoresizer.
+
+How to upgrade supported Kubernetes version
+-------------------------------------------
+
+pvc-autoresizer depends on some Kubernetes repositories like `k8s.io/client-go` and should support 3 consecutive Kubernets versions at a time.
+Here is the guide for how to upgrade the supported versions.
```

Comment:
This line is not necessary because it's clear that this section is for changing the supported Kubernetes versions by the section title.